Update vue_test_utils.json

afontcu opened the pull request:
I've added valid content on `/guide` (https://next.vue-test-utils.vuejs.org/guide/), so now tagging `guide` should work :)

shortcuts commented:
Hi @afontcu 

I've also removed the `guide` part for the `current` version. Should I add it back?

afontcu commented:
> Hi @afontcu
> 
> I've also removed the `guide` part for the `current` version. Should I add it back?

Sure, added back! Path is `/guides` and it was already indexed, but I assume it would not be displayed on a separate block from API results

shortcuts commented:
@afontcu This should be good to go, feel free to @ me if there's any issue!
